frontends/p4/fromv1.0/converters.h

```diff
@@ -78,12 +78,23 @@ class TypeConverter : public ExpressionConverter {
     explicit TypeConverter(ProgramStructure* structure) : ExpressionConverter(structure) {}
 };
 
+class ExternConverter {
```

Comment:
This is an interface that can be implemented in a backend.  It is also a (default) implementation of that interface, with the existing code that was being used for extern conversion.  Arguably there should be no default implementation (eg, it should be a pure abstract class), and the default should give an error if there are any externs in the P4_14 program.